packages/cubejs-schema-compiler/src/adapter/BaseQuery.js

```diff
@@ -2651,13 +2651,18 @@ export class BaseQuery {
 
   newSubQuery(options) {
     const QueryClass = this.constructor;
-    return new QueryClass(this.compilers, this.subQueryOptions(options));
+    return new QueryClass(this.compilers, { ...this.subQueryOptions(options), paramAllocator: undefined });
```

Comment:
We can't unset param allocator here as it should reused in case of actual `sub_query: true` members.

Reply:
What would you recommend here then, reverting the last commit and pushing forward with just the initial implementation?